### DISCUSSION TITLE
airsim.types.LidarData

### DISCUSSION THREAD

**Discussion by Sabrin0** [microsoft/AirSim#4169] [Support Q&A] [answered]
I have a question related to the `class airsim.types.LidarData`. In details, I cannot figure out what the `segmentation` value represents.
Thanks you in advance.

**Comment by jonyMarino** [comment, marked as answer]
Hi @Sabrin0. If I remember well, it is the number assigned to the mesh as explained in https://microsoft.github.io/AirSim/image_apis/#segmentation
